src/main/asciidoc/envers.adoc

```diff
@@ -0,0 +1,134 @@
+[[envers.what]]
+== What is Spring Data Envers
+
+Spring Data Envers differs from other Spring Data modules in that it is always used in combination with another Spring Data Module: Spring Data JPA.
+It makes typical Envers queries available in repositories for Spring Data JPA.
+
+== What is Envers?
+
+Envers is a Hibernate module which adds auditing capabilities to JPA entities.
+This documentation assumes you are familiar with Envers just as Spring Data Envers relies on Envers being properly configured.
+
+
+[[envers.getting-started]]
+== Getting Started
+
+As a starting point for using Spring Data Envers you need a project with Spring Data JPA.
+The easiest way to create that is using the Spring Initializr either on start.spring.io or via its integration in your favorite IDE.
+
+Now add spring-data-envers as a dependency
+[source,xml,subs="+attributes"]
+----
+<dependencies>
+
+  <!-- other dependency elements omitted -->
+
+  <dependency>
+    <groupId>org.springframework.data</groupId>
+    <artifactId>spring-data-envers</artifactId>
+    <version>{version}</version>
+  </dependency>
+
+</dependencies>
+----
+This will also bring hibernate-envers into the project as a transient dependency.
+
+Next is to make one or more repositories into {spring-data-commons-javadoc-base}/org/springframework/data/repository/history/RevisionRepository.html[`RevisionRepository`] by adding it as an extended interface.
+
+[source,java]
+----
+public interface PersonRepository
+    extends CrudRepository<Person, Long>,
+    RevisionRepository<Person, Long, Long> {}
```

Comment:
It would be nice to add callouts and explain `Person, Long, Long`. It's not obvious why there's `Long, Long`.